### PR TITLE
[ADD] Add rea to v8

### PR DESCRIPTION
**Pull request by hurrinico**

### CODE REVIEW

**Comment by tafaRU** [comment]
@hurrinico please move description to README.rst using https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as template.

Thank you in advance.

**Comment by hurrinico** [comment]
@tafaRU done

File: l10n_it_rea/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+##############################################################################
+
+from openerp import fields, models
+
+
+class resPartner(models.Model):
+    _inherit = 'res.partner'
+
+    rea_office = fields.Many2one(
+        'res.province', string='Office Province')
```
Comment (eLBati) [inline l10n_it_rea/res_partner.py]: ```res.province``` does not exist

**Comment by eLBati** [comment]
@hurrinico please move icon to static/description

File: l10n_it_rea/README.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+l10n_it_rea
+===========
+
+REA Registration
+=======================
```
Comment (tafaRU) [inline l10n_it_rea/README.rst]: Please fix equal sign number

File: l10n_it_rea/README.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+l10n_it_rea
```
Comment (tafaRU) [inline l10n_it_rea/README.rst]: Please only use friendly name instead of technical one.

**Comment by hurrinico** [comment]
@tafaRU @eLBati done

**Comment by eLBati** [comment]
:+1: thanks

**Comment by tafaRU** [comment]
LGTM :+1:

**Comment by archetipo** [comment]
:+1:

**Pull request merged**